### ISSUE TITLE
sys.flags: switch from `BoxedSysFlags` to cpythons `PyStructSequence`

### ISSUE THREAD

**Issue by undingen** [dropbox/pyston#1304]
our `BoxedSysFlags` is missing some stuff (e.g. `__str__`) I think it's better to remove it just use cpythons code for it.

**Comment by aisk** [comment]
Hi @undingen  I have a try on implement this on #1312 , can you have a look at it?

And I found these four flags is not implemented on pyston, so I just set them to `0`, I don't know is this a right way:

```cpp
int Py_DebugFlag = 0;
int _Py_QnewFlag = 0;
int Py_DontWriteBytecodeFlag = 0;
int Py_NoUserSiteDirectory = 0;
```

**Comment by kmod** [comment]
Closed by #1312
